src/components/CheckboxGroup/types.ts

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+type StringOption = {
+  key: string;
+  label: string;
+};
+
+type NodeOption = {
+  key: string;
+  label: React.ReactNode;
```

Comment:
I suggest running react codemod and cleanup all import reacts since basis is on v17
```
https://github.com/reactjs/react-codemod#update-react-imports
```